contracts/HATKlerosConnector.sol

```diff
@@ -0,0 +1,481 @@
+// SPDX-License-Identifier: MIT
+
+/**
+ *  @authors: [@unknownunknown1]
+ *  @reviewers: []
+ *  @auditors: []
+ *  @bounties: []
+ *  @deployments: []
+ */
+
+pragma solidity 0.8.16;
+
+import {IDisputeResolver, IArbitrator} from "@kleros/dispute-resolver-interface-contract/contracts/IDisputeResolver.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./HATArbitrator.sol"; // TODO: add interface
+import "./interfaces/IHATVault.sol";
+import "./interfaces/IHATKlerosConnector.sol";
+
+/**
+ *  @title HATKlerosConnector
+ *  @dev This contract acts a connector between HatsFinance and Kleros court.
+ *  This contract trusts that the Arbitrator is honest and will not reenter or modify its costs during a call.
+ *  The arbitrator must support appeal period.
+ *  The contract also trusts that HATArbitrator contract is honest and won't reenter.
+ */
+contract HATKlerosConnector is IDisputeResolver, IHATKlerosConnector, Ownable {
+
+    uint256 private constant RULING_OPTIONS = 2; // The amount of non 0 choices the arbitrator can give.
+    uint256 private constant MULTIPLIER_DIVISOR = 10000; // Divisor parameter for multipliers.        
+
+    struct DisputeStruct {
+        bytes32 claimId; // Id of the claim in HATVault contract.
+        uint256 externalDisputeId; // Id of the dispute created in Kleros court.
+        Decision ruling; // Ruling given by the arbitrator.
+        bool resolved; // True if the dispute has been resolved.
+        IHATVault vault; // Address of the vault related to a dispute.
+        Round[] rounds; // Appeal rounds.
+    }
+
+    // Round struct stores the contributions made to particular sides.
+    // - 0 side for `Decision.None`.
```

Comment:
Do we want to allow people to actively support the None option? Wouldn't it make sense to just have people support yes or no, then if it's None by the arbitrator just treat it as a no?

Reply:
apparently this is baked in in V2 (right @unknownunknown1 ?) so we just copy that logic here